cmake/OpenCVFindLibsPerf.cmake

```diff
@@ -181,7 +181,14 @@ endif(WITH_KLEIDICV)
 if(WITH_FASTCV)
   if((EXISTS ${FastCV_INCLUDE_PATH}) AND (EXISTS ${FastCV_LIB_PATH}))
     message(STATUS "Use external FastCV ${FastCV_INCLUDE_PATH}, ${FastCV_LIB_PATH}")
-    set(HAVE_FASTCV TRUE CACHE BOOL "FastCV status")
+    find_library(FASTCV_LIBRARY NAMES "fastcv"
+                 PATHS "${FastCV_LIB_PATH}" NO_DEFAULT_PATH NO_CMAKE_FIND_ROOT_PATH)
+    mark_as_advanced(FASTCV_LIBRARY)
+    if (FASTCV_LIBRARY AND (NOT "x_${FASTCV_LIBRARY}" STREQUAL "x_FASTCV_LIBRARY-NOTFOUND"))
```

Comment:
I believe we don't need special `NOTFOUND` handling - it should be built into the `if` [command](https://cmake.org/cmake/help/latest/command/if.html):

> True if the constant is 1, ON, YES, TRUE, Y, or a non-zero number (including floating point numbers). False if the constant is 0, OFF, NO, FALSE, N, IGNORE, NOTFOUND, the empty string, or ends in the suffix -NOTFOUND.